src/client/sandbox/cookie/index.ts

```diff
@@ -18,6 +18,7 @@ import {
 import { CookieRecord } from '../../../typings/cookie';
 import ChildWindowSandbox from '../child-window';
 import getTopOpenerWindow from '../../utils/get-top-opener-window';
+import { isNil } from 'lodash';
```

Comment:
Don't import the `lodash` library on the client. It causes the bundling library to the `hammerhead.js` and a lot issues with native methods in the future.